core/src/main/java/org/frankframework/soap/WsdlGenerator.java

```diff
@@ -532,7 +532,7 @@ protected void portType(XMLStreamWriter w) throws XMLStreamException {
 	}
 
 	protected String getSoapAction(IListener<?> listener) {
-		AppConstants appConstants = AppConstants.getInstance(pipeLine.getAdapter().getConfiguration().getClassLoader());
+		AppConstants appConstants = AppConstants.getInstance(pipeLine.getAdapter().getClassLoader());
```

Comment:
Dit lijkt nog weinig te helpen? Of anders een andere test die omvalt met een vergelijkbaar probleem elders.
Is de adapter in de test misschien een mock die geen ClassLoader teruggeeft maar `null`?

Reply:
Ja er valt steeds ergens anders een test om :(